.github/workflows/check-diamond-storage.yml

```diff
@@ -24,5 +24,5 @@ jobs:
 
       - name: Run Diamond Storage Check
         run: |
-          cd ./utils
-          yes | npx tsx check-diamond-storage.ts
+          yarn workspace @ubiquity/contracts build
+          forge inspect Diamond storage
```

Comment:
you should compare the output against the previous one

Reply:
I have created a new script where i compare the values before and after creating the PR